Snowflake.Data/Core/FileTransfer/StorageClient/SFGCSClient.cs

```diff
@@ -313,22 +326,35 @@ public void UploadFile(SFFileMetadata fileMetadata, byte[] fileBytes, SFEncrypti
                 return;
             }
         }
-
+        
         /// <summary>
         /// Upload the file to the GCS location.
         /// </summary>
         /// <param name="fileMetadata">The GCS file metadata.</param>
         /// <param name="fileBytes">The file bytes to upload.</param>
         /// <param name="encryptionMetadata">The encryption metadata for the header.</param>
-        public async Task UploadFileAsync(SFFileMetadata fileMetadata, byte[] fileBytes, SFEncryptionMetadata encryptionMetadata, CancellationToken cancellationToken)
+        [Obsolete("This method is deprecated. Use the method with file bytes stream instead.")]
```

Comment:
Please check the process of deprecation. At least if we have documented API, work with doc team to update that.